_posts/2023-05-31-announcing-ballerina-2201.6.0-swan-lake-update-6.md

```diff
@@ -0,0 +1,112 @@
+
+- **Persist model:** This allows you to define your data model by providing a structured representation of your data.
+
+    For example, consider the example model below.
+
+    ```ballerina
+    import ballerina/persist as _;
+    import ballerina/time;
+
+    type Employee record {|
+        readonly string id;
+        string firstName;
+        string lastName;
+        time:Date birthDate;
+        string gender;
+        time:Date hireDate;
+    |};
+    ``` 
+
+- **Persist CLI:** This generates a client API for your defined data model. 
+
+- **Persist client API:** This provides an interface to access and manipulate the data stored in your chosen data store. Whether you need to fetch data, update records, or perform other data-related operations, the persist client API simplifies the process.
+
+    The example code below demonstrates how CRUD operations can be performed on the persist model via the client API.       
+
+    ```ballerina
+    import ballerina/io;
+    import ballerina/persist;
+    import rainier.store;
+
+    store:Client sClient = check new ();
+
+    public function main() returns error? {
+        store:EmployeeInsert employee1 = {
+            id: "EMP001",
+            firstName: "John",
+            lastName: "Doe",
+            gender: "Male",
+            birthDate: {year: 1987, month: 7, day: 23},
+            hireDate: {year: 2020, month: 10, day: 10}
+        };
+
+        // Create a new employee record.
+        string[] employeeIds = check sClient->/employees.post([employee1]);
+        io:println(string `Inserted employee ids: ${employeeIds.toString()}`);
+
+
+        // Get the employee record with the ID: EMP001.
+        Employee|error employee = sClient->/employee/EMP001;
+
+        // Update the employee record with the ID: EMP001.
+        store:Employee employee = check sClient->/employees/[empId].put(
+            {hireDate: {year: 2014, month: 5, day: 1}
+        });
+
+        // Delete the employee record with the ID: EMP001.
+        store:Employee|error deleted = sClient->/employee/EMP001.delete();
+
+        // Get all the employee records.
+        stream<store:Employee, error?> employee = sClient->/employee;
+    }
+    ```
+
+In addition to the `Bal persist` feature, Swan Lake Update 6 brings a range of notable additions and improvements as described below.
+
+- **Electronic Data Interchange (EDI) support:** Swan Lake Update 6 introduces EDI message processing support in Ballerina. This capability allows the conversion of EDI messages to Ballerina records and vice versa, leveraging a specified EDI schema. Furthermore, it enables the generation of Ballerina records for EDI schemas in handling EDI data.
```

Comment:
Do we need the last sentence as the previous one said `vice versa`?

Reply:
Removed.